### PR TITLE
Hello World v2

### PR DESCRIPTION
**Pull request by giuseppe-eustaquio**
# About this PR
[BUS]69
# Types of Changes

Changed the syntax of puts
Added a new feature
* Captures and prints name
# Related PR
n/a
# Still to do
Make a website

### CODE REVIEW

File: hello_world.rb [hunk 1]
```diff
@@ -3,6 +3,7 @@
 
 puts <<~GREETING
   Hello, world!
-  How are you doing?
-  Goodbye!
+  What is your name?
 GREETING
+name = gets
+puts 'Welcome ' + name + '!'
```
Comment (colopatiron) [inline hello_world.rb:9]: Use string interpolation instead.
```suggestion
puts "Welcome, #{name}!"
```

File: hello_world.rb [hunk 2]
```diff
@@ -3,6 +3,7 @@
 
 puts <<~GREETING
   Hello, world!
-  How are you doing?
-  Goodbye!
+  What is your name?
 GREETING
+name = gets
```
Comment (colopatiron) [inline hello_world.rb]: Variable is not necessary. Put the `gets` call inside the message string.

File: hello_world.rb [hunk 3]
```diff
@@ -3,6 +3,7 @@
 
 puts <<~GREETING
```
Comment (colopatiron) [inline hello_world.rb]: Consider `print`.

**Review by colopatiron — CHANGES_REQUESTED** [review]
Left some comments.

File: hello_world.rb (hunk 3 above)
Comment (giuseppe-eustaquio) [reply hello_world.rb]: print now under consideration

**Pull request merged**